index.d.ts

```diff
@@ -72,6 +110,34 @@ declare class AmoClient {
     removeParcel(id: HexEncodedParcelId): Promise<object>
 }
 
+interface NodeConfig {
```

Comment:
This is not "node" config. It's a blockchain's config. Change it to BlockchainConfig.